README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+## biography
+### **basics**
+* Name: Hengle Li (Eric)
+* Year: first-year MAPSS
+* Concentration: Sociology
+
+#### _a few words_
+Hello, my name is Hengle Li. I also go by Eric, because there are so many various pronunciations of my first name, I don't want to bother anymore. 
+I am from China, and I went to college in UC Berkeley, as a sociology major. My research interests are social change, sociological theories, and political sociology. Bourdieu is my favorite, though I hate his writing style. After Bourdieu, I like Marx the best, because he's one of the few sociologists that know how to write properly for people to read their texts.  
+I have no programming experience before, so all these are like dark magic to me. Cheers!
+
```

Comment:
Programming is like dark magic to me too! Hopefully this course will help us become practitioners.